test/core/integration/Integration.t.sol

```diff
@@ -257,4 +268,133 @@ abstract contract Integration_Test is Base_Test {
             "settled status call return data"
         );
     }
+
+    /*//////////////////////////////////////////////////////////////////////////
+                                  MIRROR-FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    function burn(uint256 streamId) internal {
+        lockup.burn{ value: 0 }(streamId);
+    }
+
+    function cancel(uint256 streamId) internal {
+        lockup.cancel{ value: 0 }(streamId);
+    }
+
+    function cancelMultiple(uint256[] memory streamIds) internal {
+        lockup.cancelMultiple{ value: 0 }(streamIds);
+    }
+
+    function createWithDurationsLD(
+        Lockup.CreateWithDurations memory params,
+        LockupDynamic.SegmentWithDuration[] memory segmentsWithDuration
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithDurationsLD{ value: 0 }(params, segmentsWithDuration);
+    }
+
+    function createWithDurationsLL(
+        Lockup.CreateWithDurations memory params,
+        LockupLinear.UnlockAmounts memory unlockAmounts,
+        LockupLinear.Durations memory durations
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithDurationsLL{ value: 0 }(params, unlockAmounts, durations);
+    }
+
+    function createWithDurationsLT(
+        Lockup.CreateWithDurations memory params,
+        LockupTranched.TrancheWithDuration[] memory tranchesWithDuration
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithDurationsLT{ value: 0 }(params, tranchesWithDuration);
+    }
+
+    function createWithTimestampsLD(
+        Lockup.CreateWithTimestamps memory params,
+        LockupDynamic.Segment[] memory segments
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithTimestampsLD{ value: 0 }(params, segments);
+    }
+
+    function createWithTimestampsLL(
+        Lockup.CreateWithTimestamps memory params,
+        LockupLinear.UnlockAmounts memory unlockAmounts,
+        uint40 cliffTime
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithTimestampsLL{ value: 0 }(params, unlockAmounts, cliffTime);
+    }
+
+    function createWithTimestampsLT(
+        Lockup.CreateWithTimestamps memory params,
+        LockupTranched.Tranche[] memory tranches
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithTimestampsLT{ value: 0 }(params, tranches);
+    }
+
+    function renounce(uint256 streamId) internal {
+        lockup.renounce{ value: 0 }(streamId);
+    }
+
+    modifier balanceTest() {
```

Comment:
Why not put this modifier in the `Modifiers.sol` contract and override it like other examples?

Reply:
My rationale is that the modifiers placed in that file, are the BTT modifiers

Reply:
The original decision was to dump all modifiers to a single file. https://github.com/sablier-labs/v2-core/issues/1028#issuecomment-2315376028. And override whenever necessary. It was not related to BTT.